test/jdk/javax/print/CheckDupFlavor.java

```diff
@@ -26,14 +26,13 @@
  * @key printer
  * @bug 4996318 6731937
  * @summary  There should be no duplicates returned by getSupportedDocFlavors.
+ * @library /test/lib
  * @run main CheckDupFlavor
  */
 
 import javax.print.*;
-import javax.print.attribute.*;
```

Comment:
Here and later, keep the imports in place, for the same reason: targeted backportable patch.